api/gluster-volume-examples.adoc

```diff
@@ -0,0 +1,194 @@
+
+Find the list of available clusters
+
+----------
+curl -XGET http://127.0.0.1/api/1.0/GetClusterList
+----------
+
+Sample Response
+
+----------
+Status: 200 OK
+[{
+	"cluster_id": "5291c055-70d3-4450-9769-2f6fd4932afb",
+	"sds_name": "gluster",
+	"sds_version": "3.9rc1"
+}]
+----------
+
+== Cluster Flows
+
+
+Find the list of available flows for a specific cluster.
+
+----------
+curl -XGET http://127.0.0.1/api/1.0/5291c055-70d3-4450-9769-2f6fd4932afb/Flows
+----------
+
+Sample Response
+
+----------
+Status: 200 OK
+[{
+	"name": "GlusterCreateVolume",
+	"method": "POST",
+	"attributes": [{
+		"name": "Volume.volname",
+		"help": "Name of gluster volume",
+		"type": "String",
+		"default": null,
+		"required": true
+	}, {
+		"name": "Volume.bricks",
+		"help": "List of brick mnt_paths for volume",
+		"type": "List",
+		"default": null,
+		"required": true
+	}, {
+		"name": "Volume.stripe_count",
+		"help": "Stripe count of volume",
+		"type": "Integer",
+		"default": null,
+		"required": false
+	}, {
+		"name": "Volume.replica_count",
+		"help": "Replica count of volume",
+		"type": "Integer",
+		"default": null,
+		"required": false
+	}, {
+		"name": "Volume.arbiter_count",
+		"help": "Arbiter count of volume",
+		"type": "Integer",
+		"default": null,
+		"required": false
+	}, {
+		"name": "Volume.disperse_count",
+		"help": "Disperse count of volume",
+		"type": "Integer",
+		"default": null,
+		"required": false
+	}, {
+		"name": "Volume.disperse_data_count",
+		"help": "Disperse data count of volume",
+		"type": "Integer",
+		"default": null,
+		"required": false
+	}, {
+		"name": "Volume.redundancy_count",
+		"help": "Redundancy count of volume",
+		"type": "Integer",
+		"default": null,
+		"required": false
+	}, {
+		"name": "Volume.transport",
+		"help": "Transport type for volume",
+		"type": "String",
+		"default": null,
+		"required": false
+	}, {
+		"name": "Volume.force",
+		"help": "If force execute the action",
+		"type": "Boolean",
+		"default": null,
+		"required": false
+	}]
+}, {
+	"name": "GlusterDeleteVolume",
+	"method": "DELETE",
+	"attributes": [{
+		"name": "Volume.volname",
+		"help": "Name of gluster volume",
+		"type": "String",
+		"default": null,
+		"required": true
+	}, {
+		"name": "Volume.vol_id",
+		"help": "ID of the gluster volume",
+		"type": "String",
+		"default": null,
+		"required": true
+	}]
+}]
+----------
+
+=== List Volumes
+
+Sample Request
+
+----------
+curl -XGET http://127.0.0.1/api/1.0/5291c055-70d3-4450-9769-2f6fd4932afb/GetVol
+umeList
+----------
+
+Sample Response
+
+----------
+Status: 200 OK
+
+[{
+	"cluster_id": "5291c055-70d3-4450-9769-2f6fd4932afb",
+	"deleted": "",
+	"name": "vol1",
+	"status": "Stopped",
+	"vol_id": "80e98588-176c-4220-b41c-e95cd7137aff",
+	"vol_type": "Distribute",
+	"mount_opts": "",
+	"path": "dhcp-1.lab.tendrl.example:/root/bricks/vol1_b2",
+	"port": "49152",
+	"fs_type": "",
+	"hostname": "dhcp-1.lab.tendrl.example",
+	"brick_count": "2"
```

Comment:
Brick count is already in list of bricks. If you would like to provide this info, I think it will be better to do that with receipt how to count that number, like in ```gluster volume info```:
```2 x 2 x (2 + 1) = 12```